devices/tuya.js

```diff
@@ -4279,4 +4279,208 @@ module.exports = [
         toZigbee: [],
         configure: tuya.configureMagicPacket,
     },
+    {
+        fingerprint: [{
+            modelID: 'TS0001',
+            manufacturerName: '_TZ3000_bmqxalil',
+        }],
+        model: 'TS0001_switch_1_gang',
+        vendor: 'TuYa',
+        description: '1-Gang switch with backlight',
+        fromZigbee: [tuya.fz.backlight_mode_off_on, fz.on_off, tuya.fz.power_on_behavior_2],
+        toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+        exposes: [
+            e.switch(),
+            e.power_on_behavior(),
+            exposes.binary('backlight_mode', ea.ALL, 'on', 'off').withDescription('Backlight Mode'),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
```

Comment:
`fromZigbee`, `exposes` and `toZigbee` can be removed and be replaced with: `extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true}),`

Note this will require the latest dev branch (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

Reply:
found out that backlightModeOffOn is not working properly, reason for that is that `expose` of the `extend` creates the binary switch with ON/OFF while converters are expecting lower case, so I'm using the manual setup of exposing binary switch while using the available converters

hope it make sense, didn't want to change the expose as I'm not sure how it will effect other devices,
most likely the right approach will be to support within the converter both upper and lower cases.

Reply:
> while converters are expecting lower case,

The converters works with both casing due to:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/b49ae8189b2bc28c0089fdd011b635c370049335/lib/tuya.js#L1590

Reply:
It's not working (sending the command), switching to my approach solved it

Reply:
Thinking about it the problem is with the from zigbee not running the command, but getting the result from zigbee

Reply:
You are right, fixed! https://github.com/Koenkk/zigbee-herdsman-converters/commit/59208f8e6acc4bde3b346f8f5b2b8b25e870586d

Reply:
great, thanks